Tests/Functional/StepProviders/CsvConfigurableStepsProviderTest.php

```diff
@@ -45,22 +45,22 @@ public static function setUpBeforeClass()
     {
         parent::setUpBeforeClass();
         //create a temporary folder
-        mkdir(self::TMP_FOLDER);
+        \mkdir(self::TMP_FOLDER);
     }
 
     /**
      * {@inheritdoc}
      */
     public static function tearDownAfterClass()
     {
-        $files = glob(self::TMP_FOLDER.'*');
+        $files = \glob(self::TMP_FOLDER.'*');
```

Comment:
Even though is not really related to the ticket, using https://symfony.com/doc/current/components/filesystem.html#remove would be nicer here

Reply:
yes it would be nicer.
we would have to add it as a dependancy. Do we wana do that as part of this pull?

Reply:
We require symfony/symfony (which is not especially a good idea) already, which include the FS component: https://packagist.org/packages/symfony/symfony#v2.8.49